pyvisa/cmd_line_tools.py

```diff
@@ -10,22 +10,17 @@
 
 import argparse
 import logging
-from typing import Optional
 
 
-def visa_main(command: Optional[str] = None) -> None:
-    """Run the main entry point for command line tools.
-
-    Parameters
-    ----------
-    command : str, optional
-        What command to invoke, if None the value is read from the command
-        line arguments
+def get_shell_parser() -> argparse.ArgumentParser:
+    """Create argument parser for pyvisa-shell program."""
 
+    description = """
+    PyVISA interactive console for debugging and testing with PyVISA-compatible
+    devices.
     """
```

Comment:
I'm not myself familiar enough with the tool to provide a concise description. Any suggestions on the topic are very welcome.